### ISSUE TITLE
suggestion/request, close current tab when opening rerouted page

### ISSUE THREAD

**Issue by RonaldRegister** [gnormandin/Archive-Page#3]
not sure is you want suggestions or requests and if this is the right place?

Would it be possible to close the current tab when clicking the extension icon? So upon clicking, open current page on archive.today, close current tab, switch to next tab. 

Would also be great for the iOS version.

If you'd need a tester, I'll be happy to assist.

**Comment by gnormandin** [comment]
Not sure closing the current tab would be preferred behavior, as the user would lose tab history in the previous tab. 

Would opening the archive.today link in the current tab be a good compromise?

The original Firefox extension has this as configurable, but background pages are not supported on iOS from what I see, and this is needed for the popup configuration window.

I may be wrong though, and I will explore further.

**Comment by RonaldRegister** [comment]
> Would opening the archive.today link in the current tab be a good compromise?

of course! I assumed this wasn't possible, and the plugin opened in a new tab for a reason. Wouldn't be a compromise, it would be perfect.

**Comment by gnormandin** [comment]
Just wanted to say I'm still working on this in my spare time... I wanted to maintain the configurability of the original extension and so I'm working on making this flexible. 

Will let you know when I have more.
